### ISSUE TITLE
API changes to "Dice So Nice" causes rolls to not work

### ISSUE THREAD

**Issue by Javalar** [RedReign/FoundryVTT-BetterRolls5e#58]
The "Dice so Nice!" module has made changes to its API which breaks Better Rolls. The call to `game.dice3d.showForRoll` needs to be changed to:

```
game.dice3d.showForRoll(customRoll.dicePool, game.user, wd.whisper, wd.blind || false).then(...
```

**Comment by RedReign** [comment]
Bad timing - I just pushed a fix for this in v1.1.6! Thank you for mentioning though

**Comment by Javalar** [comment]
Nice, thanks.

And only now do I notice my fix wasn't even correct. 🤪
